eravm-stable-interface/src/state_interface.rs

```diff
@@ -82,6 +83,8 @@ pub trait CallframeInterface {
     fn set_aux_heap_bound(&mut self, value: u32);
 
     fn read_code_page(&self, slot: u16) -> U256;
+
+    fn last_precompile_cycles(&self) -> usize;
```

Comment:
This is probably unused, as the integration uses the specific precompile cycle numbers instead.